src/lib/scene.ts

```diff
@@ -14,6 +15,10 @@ export class Scene extends IPSODevice {
 	@deserializeWith(obj => new LightSetting().parse(obj))
 	public lightSettings: LightSetting[] = [];
 
+	@ipsoKey("15015") /// ??? (guessed ID, does not work, how to get this data? Plugs can be part of a Scene.
```

Comment:
The current IKEA android app (v1.7.0) does not yet have a key for `plugSettings` and no property on the `Scene` class, so I guess this won't work for now.

Reply:
Strange thing is you can add plugs in scenes, so how does the app communicate this with the gateway?

Reply:
Are you using the iOS app or the Android one? They might have different capabilities - or I did not download the most recent APK.

Reply:
I double-checked - 1.7.0 is the most recent android version. I cannot find any reference to plugs in scenes, but most of the code is badly obfuscated anyways. It might be the updates to plugs are indeed communicated as part of the `lightSetting`s, since they have some common properties (`onOff`, `dimmer`).

Reply:
I've checked, they are not included in the lightSettings, but maybe you need to ask for them by subscribing the endpoints. It not fully clear to me yet what is communicated anyway and what you need to ask for (I need to catch up on Coap anyway). But I did check on an iPhone, I will check on an Android phone this weekend.

Reply:
The protocol is basically a (very simplified) HTTPS-over-UDP with JSON payloads. In order to send commands, you don't need to subscribe to (or rather "observe") anything.  Observing a resource results in the server pushing changes to you, instead of you having to re-fetch the resource every now and then.

I've reverse-engineered most commands by looking at the app, try-and-error, reading other developers' work and sniffing the traffic. Since I don't have any plugs (yet), the last part is currently not an option.

> I've checked, they are not included in the lightSettings

So you added a `plugSetting` to the `lightSettings` array of a command and that didn't do anything, correct?

Reply:
Didn't try a command yet. Just looked what I got back when I changed a Scene containing a plug while observing groups and scenes. Only the lights data seems to come back also looked in the debugger into the JSON payload, it doesn't contain any other tags.

(NB I first tried 15014  and see I left 15015 in but both were guessed. )

Reply:
I see... for which endpoints did you look at the responses? We probably need to observe the plugs too.

Reply:
I will have to look at this tonight...